src/Pact/Types/Type.hs

```diff
@@ -332,7 +375,9 @@ isVarTy :: Type v -> Bool
 isVarTy TyVar {} = True
 isVarTy _ = False
 
--- | a `canUnifyWith` b means a "can represent/contains" b
+-- | Used by runtime typechecking to unify a specification with a value,
+-- 'a canUnifyWith b' means 'a' "can represent/contains" 'b'.
+-- In use, 'a' is the spec, and 'b' is the value type being unified with 'a'.
```

Comment:
Clarified this comment by looking at usage: the first arg is always the "spec"/declared type and the second arg is always the "value". This is also where I realized that the "ModSpec" name may be pedantic, because the second arg isn't a "spec" ... but it is a `Type`.